### DISCUSSION TITLE
Actividad 0 - Baroni Rodrigo, Mauro

### DISCUSSION THREAD

**Discussion by MauroBaroni** [frlp-utn-ingsoft/catedra#45] [Show and tell]
https://github.com/MauroBaroni/IngSoftwareS41Baroni

**Comment by RodrigoJacznik** [comment]
Buenas.
Los commits y sus mensajes estan correctos. Pero cuando entro a una `poll`, esta no tiene `choice`, te olvidaste de pushearlo?
Una poll se tendría que ver así:
![image](https://github.com/frlp-utn-ingsoft/catedra/assets/2715743/97097738-27ad-40d5-823a-ec0316de2a58)
Creo que te falto la ultima parte de esta sección https://docs.djangoproject.com/en/5.0/intro/tutorial02/#playing-with-the-api, la que esta justo antes de "Introducing the Django Admin".

En la parte 7 (https://docs.djangoproject.com/en/5.0/intro/tutorial07/) se muestra como modificar el admin para agregar choice a una poll. Esta parte no es obligatorio que lo hagas pero quizás es mas fácil que agregar las choice mediante la shell de django.

**Reply by MauroBaroni** [reply]
me faltaba, agrege la pregunta y las opciones con estos pasos https://docs.djangoproject.com/en/5.0/intro/tutorial02/#playing-with-the-api ya lo subi al repositorio

**Reply by RodrigoJacznik** [reply]
Ahora si 👍 
Actividad aprobada.
